test/integration/platform_test.go

```diff
@@ -28,6 +30,12 @@ func TestPlatformSelection(t *testing.T) {
 			os:             "linux",
 			expectedDigest: "sha256:1ee006886991ad4689838d3a288e0dd3fd29b70e276622f16b67a8922831a853", // direct from repo manifest
 		},
+		{
```

Comment:
not: should we add a `name` field to these tests to give some context beyond that we're testing different combinations of arch/os

I think both of us took at look at this earlier and were confused or unsure about the comments `// from generated manifest` vs `direct from repo manifest`and what they added to the case

Reply:
We added a top-level comment explaining how this test works instead.